js/app/controllers/content-product.ng

```diff
@@ -138,7 +138,158 @@ mkdir my-new-bucket</pre>
       the Cloud Tools for PowerShell module via <code>Import-Module GoogleCloud</code>.
     </p>
   </div>
+  
+    <!-- Google Cloud BigQuery -->
+  <div ng-switch-when="google-cloud-bigquery">
+    <p>
+      Google BigQuery is a versatile tool that solves the problem of storing and querying massive 
+      datasets without having to worry about data formats, underlying resources, and other things 
+      that distract you from your analysis.
+    </p>
+    
+    <h2>Dataset</h2>
+    <p>
+      To use BigQuery in a Cloud project, first create a <em>Dataset</em> using the 
+      <cmdlet-ref name="New-BqDataset"></cmdlet-ref> cmdlet. This will take in basic information and 
+      create the resource serverside. Locally, a reference Dataset object is returned. To get a reference
+      object for an existing dataset, use <cmdlet-ref name="Get-BqDataset"></cmdlet-ref>.
+    </p>
+    <pre>
+# Makes a new dataset with DatasetId "my_dataset".
+$dataset = New-BqDataset "my_dataset" -Name "Test Dataset" -Description "Some test data for BigQuery"
+
+# The following two lines do the same thing.
```

Comment:
"do the same thing" isn't useful, since you don't explain what they do.

````
# Two ways to get a dataset: by name or by existing Dataset object.
````